Sources/SRGDataProviderCombine/RadioServices.swift

```diff
@@ -152,6 +152,14 @@ public extension SRGDataProvider {
         return paginatedObjectsTriggeredPublisher(at: Page(request: request, size: pageSize), rootKey: "showList", type: SRGShow.self, paginatedBy: signal)
     }
     
+    /**
+     *  All radio shows alphabetically.
+     */
+    func allRadioShows(for vendor: SRGVendor) -> AnyPublisher<[SRGShow], Error> {
```

Comment:
I would suggest simply:

```suggestion
    func radioShows(for vendor: SRGVendor) -> AnyPublisher<[SRGShow], Error> {
```

since the per-channel variant requires a mandatory `channelUid` parameter anyway.